python/MOCK_sensors.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 import numpy as np
 import rospy
+import random
```

Comment:
Let's use the `numpy` equivalents of the `random` functions that you used to save an import